sdk/servicebus/azure-mgmt-servicebus/README.md

```diff
@@ -1,12 +1,13 @@
 # Microsoft Azure SDK for Python
 
 This is the Microsoft Azure Service Bus Management Client Library.
-This package has been tested with Python 3.6+.
+This package has been tested with Python 3.7+.
 For a more complete view of Azure libraries, see the [azure sdk python release](https://aka.ms/azsdk/python/all).
 
 ## _Disclaimer_
 
 _Azure SDK Python packages support for Python 2.7 has ended 01 January 2022. For more information and questions, please refer to https://github.com/Azure/azure-sdk-for-python/issues/20691_
+_Python 3.7 or later is required to use this package. For more details, please refer to [Azure SDK for Python version support policy](https://github.com/Azure/azure-sdk-for-python/wiki/Azure-SDKs-Python-version-support-policy)._
```

Comment:
(FYI: We actually want to keep the message that we have about the end of Python 2.7 support.

If you take a look at the issue we refer to here (https://github.com/Azure/azure-sdk-for-python/issues/20691), you can see that we're announcing the end of support for Python 2.7. We don't usually have big announcements like this for dropping Python 3.x versions, b/c Python 2.7 is a specific/special case. You can read more about this [[here](https://www.python.org/doc/sunset-python-2/)].)